### PR TITLE
Bootstrap issue

### LINKED ISSUE #301: Time between native protocol is enabled and node is started matters
## Bug Report

When a node is live it's considered as ready and the next node is started, however the bootstrap of the next node will fail cause native protocol is not enable yet. 

**What did you do?**
Create a new cluster with 2 nodes and check if nodes see each other or not. It happens that they belong to different rings.

**What did you expect to see?**
the bootstrap image tries to access the native port of seed nodes to know if they can join an existing ring. But as a node is ready before it can happen that they don't see each other.

**What did you see instead? Under which circumstances?**
nodes do not see each other

**Environment**
* casskop version: 1.1.2

**Possible Solution**
Test the JMX port instead of the native port.

### PR DESCRIPTION
**Pull request by cscetbon**
| Q               | A
| --------------- | ---
| Bug fix?        | [X]
| Related tickets | fixes #301 
| License         | Apache 2.0

### What's in this PR?

Test Jolokia port instead of native port when attempting to bootstrap nodes

### Why?

 It prevents having nodes being created in separated clusters

### CODE REVIEW

File: samples/cassandracluster.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   bootstrapImage: orangeopensource/cassandra-bootstrap:0.1.4
```
Comment (fdehay) [inline samples/cassandracluster.yaml:9]: why not 0.1.8?

File: samples/cassandracluster.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ spec:
...
   configMapName: cassandra-configmap-v1
   dataCapacity: "200Mi"
-  dataStorageClass: "local-storage"
+  dataStorageClass: local-path
```
Comment (fdehay) [inline samples/cassandracluster.yaml:12]: WHere does this local-path come from? Is it an old update?

**Comment by fdehay** [comment]
Will CCL need to change their network policies for this to work on Rick?
Or is port 9042 open as well already?

File: samples/cassandracluster.yaml (hunk 2 above)
Comment (cscetbon) [reply samples/cassandracluster.yaml:12]: K3d

**Comment by cscetbon** [comment]
> Will CCL need to change their network policies for this to work on Rick?
> Or is port 9042 open as well already?
They’ll have to update their policies

**Review by fdehay — DISMISSED** [review]

**Pull request merged**